extension/src/CSV/CSV.ts

```diff
@@ -42,6 +43,17 @@ export class CSV {
     return headerMap;
   }
 
+  public get headers(): string[] {
+    return this._headers;
+  }
+
+  public set headers(headers: string[]) {
+    this._headers = [
+      ...this._headers,
+      ...headers.filter((h) => this._headers.indexOf(h) === -1),
+    ];
+  }
```

Comment:
This function took some time to understand its purpose... But after a while I think I understand. 😀